powersimdata/input/converter/pypsa_to_grid.py

```diff
@@ -432,7 +437,8 @@ def _translate_pnl(self, pnl, key):
         data frames.
 
         :param str pnl: name of the time-dependent dataframe.
-        :param str key: key in the :data:`pypsa_const` dictionary.
+        :param str key: key in :data:`powersimdata.input.const.pypsa_const.pypsa.const`
```

Comment:
Should this be `powersimdata.input.const.pypsa_const.pypsa_const`?

Reply:
Done